app/utils/socket.js

```diff
@@ -209,15 +209,20 @@ export default Ember.Object.extend(Ember.Evented, {
       this.set('_disconnectedAt', (new Date()).getTime());
     }
 
-    if ( isSafari && !wasConnected && this.get('url').indexOf('wss://') === 0 )
+    if ( !warningShown && !wasConnected )
     {
       this.set('autoReconnect', false);
       this.set('_state', DISCONNECTED);
-      if ( !safariWarningShown )
+
+      const intl = window.l('service:intl');
+      let warningMessage = intl.t('growl.webSocket.connecting.proxyIssueWarning');
+      if ( isSafari && this.get('url').indexOf('wss://') === 0 )
       {
-        safariWarningShown = true;
-        window.l('service:growl').error('Error connecting to WebSocket','Safari does not support WebSockets connecting to a self-signed or unrecognized certificate.  Use http:// instead of https:// or reconfigure the server with a valid certificate from a recognized certificate authority.  Streaming stats, logs, shell/console, and auto-updating of the state of resources in the UI will not work until this is resolved.');
+        warningMessage += intl.t('growl.webSocket.connecting.safariCertWarning');
       }
+      warningMessage += intl.t('growl.webSocket.connecting.warning');
```

Comment:
Combine proxyissuewarning and warning (see translation file below) with the optional safari message below it